src/community/design-system-day-2024/speaker-information.md

```diff
@@ -29,11 +29,11 @@ Zainab Daodu is a seasoned Software Engineer, Technical Writer and founder of Wr
 
 ### Leyla Kee-McParlin, Department for Environment, Food & Rural Affairs
 
-Leyla is a Lead Content Designer at Defra, directing a team working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in the Liverpool City Region.
+Leyla is a Lead Content Designer at DEFRA, directing a team working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in the Liverpool City Region.
```

Comment:
`Leyla is a Lead Content Designer at Defra, working on borders and trade services. She's a service assessor, a Civil Service Social Mobility Ambassador, and does voluntary work with schools in Liverpool.`